ui/StatusQ/src/wallet/managetokenscontroller.cpp

```diff
@@ -365,7 +368,12 @@ bool ManageTokensController::lessThan(const QString& lhsSymbol, const QString& r
     leftPos = leftVisible ? leftPos : INT_MAX;
     rightPos = rightVisible ? rightPos : INT_MAX;
 
-    return leftPos <= rightPos;
+    if (leftPos < rightPos)
+        return -1;
+    else if (leftPos > rightPos)
+        return 1;
+    else
+        return 0;
```

Comment:
https://en.cppreference.com/w/cpp/utility/compare/compare_three_way

But I guess we can't get all the goodies, since it's C++ 20 :/